recipes/README.md

```diff
@@ -27,9 +27,9 @@ Recipes in torchtune are:
 
 &nbsp;
 
-### Architecture Optimization
+### Quantization and Sparsity
 
-torchtune integrates with `torchao`(https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see `recipes/configs/quantization.yaml` for more details.
+torchtune integrates with `torchao`(https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) for more details.
```

Comment:
Why keep torchao as code and not just directly hyperlink here?

Reply:
oh yeh good point